wslpy/exec.py

```diff
@@ -13,6 +49,7 @@ def cmd(command):
     command : str
         string of `cmd.exe` commands.
     """
+    preCheck()

```

Comment:
I am not against this but using `assert` would be a better idea. We don't want to exit with no error code returned; it would be a bad practice to do so. Same as below.

Reply:
is the preCheckAssert method the correct implementation ?

Reply:
I would suggest just use it directly, no need for a separate function

Reply:
So I should eliminate preCheck function only right ?? isInteropEnabled() is okay ??

Reply:
Yes